client/src/main/java/uk/co/blackpepper/bowman/ObjectMapperConfigurer.java

```diff
@@ -0,0 +1,14 @@
+package uk.co.blackpepper.bowman;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Interface supporting the configuration of the Jackson {@link com.fasterxml.jackson.databind.ObjectMapper}
+ * used internally by the {@link Client} and the RestOperations.
```

Comment:
Please just drop the `and the RestOperations` from this comment - as RestOperation is private API, it has no place in Javadoc. Thanks!

Reply:
Changed